### ISSUE TITLE
Change camera info using yaml file

### ISSUE THREAD

**Issue by manuelgitgomes** [miguelriemoliveira/rosbag_tools#5]
Changing camera info from a yaml file, instead of using argparse

**Comment by manuelgitgomes** [comment]
Hello @miguelriemoliveira

This issue is regarding lardemua/atom#935, in which I needed to change the intrinsic parameters of the cameras in the bag file.
This was a function already available on this repo, however, it used arguments from argparse, instead of the official yaml files.
To achieve this, I created a folder named `src` to host dependencies, on which I placed this previously created function:
https://github.com/miguelriemoliveira/rosbag_tools/blob/99f5e859f3ad2c0bd6281fcec7e1653586d94d1e/src/rosbag_tools/config_camera_intrinsics.py#L9-L97

This function is now called from the old file `config_camera_intrinsics` and the new `config_camera_intrinsics_yaml`.

This a change in repository organization, so this is in a separate branch.
If you agree, I will merge it.
